doc/nrf/protocols/matter/overview/bridge.rst

```diff
@@ -0,0 +1,67 @@
+
+Additionally, the bridge represents every bridged device using one or more Matter endpoints, depending on the functionalities supported by the bridged device.
+Every bridged device exposed by the bridge has to fulfill all requirements for the corresponding Matter device type.
+Unlike native Matter nodes, the bridged device has a dedicated Bridged Device Basic Information cluster enabled on its endpoint to provide basic information about the device.
+
+Since bridged devices can be added to the bridge or removed from it at runtime, the list of endpoints exposed by the bridge can be dynamically updated.
+
+Implementation in the |NCS|
+***************************
+
+The |NCS| includes a reference implementation of the Matter bridge core, located in the :file:`samples/matter/common/bridge` directory.
+The core implementation is not dependent on any specific connectivity technology, and provides basic bridge functionalities and a generic API.
+In addition to the core part, the |NCS| also includes an example :ref:`matter_bridge_app` application that you can use as a reference for creating your own application.
+
+The Matter bridge translates between the Matter data model and the data representation specific for other protocols.
+It is not possible to provide a generic translation between Matter and all other IoT protocols.
+The |NCS| application implements supports for the Bluetooth LE protocol, but you can also disable Bluetooth LE support and use the core implementation part to create an application bridging Matter to any connectivity technology.
+
+The bridge application can be customized in various ways depending on the supported connectivity protocol and Matter device types for bridging, or the UI used to control the device.
+The following figure shows the software components of the bridge core and their relation to the user's application:
+
+.. figure:: images/matter_bridge_architecture.svg
+   :alt: Matter bridge software architecture
+
+   Matter bridge software architecture
+
+In the figure, there are four major software modules included in the Bridge core:
+
+* ``Matter Bridged Device`` - The abstract class that defines common functionalities for all Matter bridged devices.
+  The user's application can create specific Matter bridged device objects, such as OnOff Light Device, that inherit the base class functionality.
+* ``Bridged Device Data Provider`` - The abstract class that defines common functionalities for all Bridged Device Data Providers.
+  The providers are objects that provide data for Matter bridged devices and represents the interface to the bridged device on the non-Matter side.
+  The user's application can create a specific implementation of a data provider that can, for example, obtain data from physical devices using the Bluetooth LE protocol.
+* ``Bridge Manager`` - The main object of a bridge.
```

Comment:
`Main object` sounds a bit odd to me, maybe it's better to refer to it as a `core component/module`

Reply:
But we described bridge "core", as a group of 4 modules common for all bridge applications, no matter what technologies they use. Would replacing it with principal/chief word work for you?

Reply:
I agree using "core" for two different meanings on the same page isn't great. Would "main component" work?

Reply:
I updated it with the "main component".